RosBE-Windows/Powershell/RosBE.ps1

```diff
@@ -69,7 +69,7 @@ $global:_BUILDBOT_SVNSKIPMAINTRUNK = "0"
 $ENV:CCACHE_SLOPPINESS = "time_macros"
 
 # Fix Bison package path (just in case RosBE is installed in a path which contains spaces)
-$ENV:BISON_PKGDATADIR = ((New-Object -ComObject Scripting.FileSystemObject).GetFolder("$_ROSBE_HOST_MINGWPATH\share\bison")).ShortPath
+$ENV:BISON_PKGDATADIR = ((New-Object -ComObject Scripting.FileSystemObject).GetFolder("$_ROSBE_BASEDIR\bin\data")).ShortPath
```

Comment:
With my Bison 3.5 binary from https://svn.reactos.org/RosBE-Temp/RosBE-2.2_Dec2019.7z, this environment variable is required and the data path is share\bison.
We also need another environment variable "M4" with the full path to the m4.exe binary (which is also shipped in a defined version in RosBE-2.2_Dec2019.7z).

Reply:
Fixed I hope.